Formula/z/zipkin.rb

```diff
@@ -0,0 +1,30 @@
+class Zipkin < Formula
+  desc "Collect and visualize traces written in Zipkin format"
+  homepage "https://zipkin.io"
+  url "https://search.maven.org/remotecontent?filepath=io/zipkin/zipkin-server/2.27.1/zipkin-server-2.27.1-exec.jar"
+  sha256 "19ad7268c5cd9fdae7876bbfeceb8ddba4675fc7ab752e5e37177bc46335ac1f"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://search.maven.org/remotecontent?filepath=/io/zipkin/zipkin-server/maven-metadata.xml"
+    regex(%r{<version>v?(\d+(?:\.\d+)+)</version>}i)
+  end
+
+  depends_on "openjdk"
+
+  def install
+    (libexec/"bin").install "zipkin-server-#{version}-exec.jar"
+    bin.write_jar_script libexec/"bin/zipkin-server-#{version}-exec.jar", "zipkin"
+  end
+
```

Comment:
we can also add `service` block as well using the default config?

Reply:
ok do I need to do `keep_alive true`? it is a daemon..

Reply:
yeah, something like that and log setup. you can find examples in the codebase

Reply:
ok will look around, thanks for the tips!